databases/week2/p4_creating_database.sql

```diff
@@ -0,0 +1,45 @@
+CREATE DATABASE ART_WORK_GALLERY
```

Comment:
I like the creativity, especially with the sales table. Maybe in the diagram you can remove the connection between employee and artwork since they're not directly referencing each other